### PR TITLE
Update UI

### PR DESCRIPTION
**Pull request by perplexingcode**

### CODE REVIEW

File: src/app/[lng]/cart/page.js [hunk 1]
```diff
@@ -164,9 +164,8 @@ export default function Cart({ params: { lng } }) {
           </>
         )}
       </div>
-
-      {cartData && cartData?.items?.length != 0 && (
-        <div className="w-full max-w-[700px] fixed bottom-[90px] md:bottom-[20px] h-[50px] border-t-[1px] md:border-[1px] border-gray-600 p-2 bg-green-600 text-white md:rounded-md">
+      {cartData?.items?.length != 0 && (
```
Comment (vietanhdev) [inline src/app/[lng]/cart/page.js:167]: Adding `{cartData && ` will help to hide the button when the cart is empty.

File: src/app/[lng]/products/page.js [hunk 2]
```diff
@@ -48,7 +48,11 @@ export default function ProductManagement({ params: { lng } }) {
         )}
       </div>
       <div
+<<<<<<< HEAD
```
Comment (vietanhdev) [inline src/app/[lng]/products/page.js:51]: Fix code merging. Please build before committing:
```
npm run build
```

File: src/app/[lng]/stores/page.js [hunk 3]
```diff
@@ -52,7 +52,11 @@ export default function StoreManagement({ params: { lng } }) {
         )}
       </div>
       <div
+<<<<<<< HEAD
```
Comment (vietanhdev) [inline src/app/[lng]/stores/page.js:55]: Fix code merging

**Review by vietanhdev — CHANGES_REQUESTED** [review]
Run the following command to format the code before committing.
(We will setup this into git hook later).
```
npm run pr
```

**Comment by vietanhdev** [comment]
There are some changes that need to be added:
- Fix the product skeletons (too small now).
- Improve responsiveness.
image

**Review by haitt00 — APPROVED** [review]

**Review by haitt00 — COMMENTED** [review]
Screenshot 2023-07-11 at 13 01 48
@perplexingcode Button lưu thay đổi: boundary needs fixing.

Please resolve requested changes when done 👯

**Review by haitt00 — APPROVED** [review]

**Pull request merged**